guides/release-notes.md

```diff
@@ -7,6 +7,16 @@ We love hearing your feedback! You can help us improve the docs by reporting any
 ![Was this helpful, yes/no buttons](../media/feedback.png)
 
 ---
+### Updated API: Wix Billing Backend
+
+`wix-billing-backend`
+
+We updated the following functions in the [Invoices API](https://www.wix.com/velo/reference/wix-billing-backend/invoices), of the[`wix-billing-backend`](https://www.wix.com/velo/reference/wix-billing-backend/events) module.
```

Comment:
Lead with a statement that we introduced a change to the locale object, and then go into what functions it affects.